eventsourcing/src/main/scala/com/evolutiongaming/akkaeffect/eventsourcing/Engine.scala

```diff
@@ -394,6 +427,7 @@ object Engine {
                           val persist =                                        // create {{{ persist: F[Unit] }}} job
                             for {
                               seqNr <- close { append(change.events) }.attempt // persist events if [[close]] allow
+                              _     <- eff.set(state1)                         // update effective state
```

Comment:
do I read this correctly? it will `eff.set(state1)` also in case of error

Reply:
thanks!